packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -1217,36 +1229,21 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     return person && person.personImage ? person.personImage : null;
   }
 
-  private hovered(icon: string, state: boolean) {
-    switch (icon) {
-      case 'email':
-        this.isEmailHovered = state;
-        break;
-      case 'chat':
-        this.isChatHovered = state;
-        break;
-      case 'video':
-        this.isVideoHovered = state;
-        break;
-      case 'call':
-        this.isCallHovered = state;
-        break;
-    }
-  }
+  @internalProperty() private hoverStates: Record<HoverStatesActions, boolean> = {
+    email: false,
+    chat: false,
+    video: false,
+    call: false
+  };
 
-  private handleMouseEnter(icon: string) {
-    clearTimeout(this._mouseEnterTimeout);
-    clearTimeout(this._mouseLeaveTimeout);
-    this.hovered(icon, true);
-    this._mouseEnterTimeout = setTimeout(this.hovered.bind(this), 500);
-  }
+  private setHoveredState = (icon: string, state: boolean) => {
+    this.hoverStates[icon] = state;
+    this.hoverStates = { ...this.hoverStates };
```

Comment:
Can we please add a comment in here to explain that the assignment to the @internalProperty decorated hoverStates property is what triggers the re-render to explain why we're doing what appears to be a redundant thing here?